socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,27 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
```

Comment:
Actually, the ``PolyStorageError`` object is always used, with this line: ``if storage_exception.has_exceptions():``. So it does make sense to create it every time, and it would make the code really a lot more complex for almost nothing to make it ``None`` at first. 

So, I'm not going to change this. :)